$OpenBSD: patch-include_tdb_h,v 1.1 2015/03/29 12:28:21 ajacoutot Exp $

Unbreak header by defining _PUBLIC_
XXX: newer version just removed _PUBLIC_, remove that patch on tdb upgrade

--- include/tdb.h.orig	Sun Mar 29 14:14:18 2015
+++ include/tdb.h	Sun Mar 29 14:14:52 2015
@@ -52,6 +52,10 @@ extern "C" {
 #define TDB_DISALLOW_NESTING 1024 /* Disallow transactions to nest */
 #define TDB_INCOMPATIBLE_HASH 2048 /* Better hashing: can't be opened by tdb < 1.2.6. */
 
+#ifndef _PUBLIC_
+#define _PUBLIC_
+#endif
+
 /* error codes */
 enum TDB_ERROR {TDB_SUCCESS=0, TDB_ERR_CORRUPT, TDB_ERR_IO, TDB_ERR_LOCK, 
 		TDB_ERR_OOM, TDB_ERR_EXISTS, TDB_ERR_NOLOCK, TDB_ERR_LOCK_TIMEOUT,
